kratos/utilities/time_discretization.cpp

```diff
@@ -0,0 +1,122 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Philipp Bucher
+//
+
+// System includes
+#include <limits>
+
+// External includes
+
+// Project includes
+#include "includes/checks.h"
+#include "time_discretization.h"
+
+namespace Kratos {
+namespace TimeDiscretization {
+
+void BDF1::ComputeBDFCoefficients(const double DeltaTime, std::array<double, 2>& rCoefficients) const
+{
+    KRATOS_ERROR_IF(DeltaTime < std::numeric_limits<double>::epsilon())
+        << "Expects DeltaTime > 0!" << std::endl;
+
+    rCoefficients[0] =  1.0/DeltaTime;
+    rCoefficients[1] = -1.0/DeltaTime;
+}
+
+void BDF2::ComputeBDFCoefficients(const double DeltaTime, std::array<double, 3>& rCoefficients) const
+{
+    KRATOS_ERROR_IF(DeltaTime < std::numeric_limits<double>::epsilon())
+        << "Expects DeltaTime > 0!" << std::endl;
+
+    const double denom = 1.0/(2.0*DeltaTime);
+
+    rCoefficients[0] =  3.0 / denom;
```

Comment:
```suggestion
    rCoefficients[0] =  3.0 * denom;
```
We should probably add some simple tests.

Reply:
> I don't know about whether the BDF3-6 can allow for variable time steps

No clue either ...
@loumalouomega do you know?

Anyway, I think this could be easly added by adding the functions